hack/cmd_util.sh

```diff
@@ -31,6 +31,19 @@ function os::cmd::expect_success_and_text() {
 	os::cmd::internal::expect_exit_code_run_grep "${cmd}" "os::cmd::internal::success_func" "${expected_text}"
 }
 
+# expect_success_and_multiple_text runs the cmd and expects an exit code of 0
+# as well as running a grep test to find the given strings in the output
+# TODO: Run the command once and not for every string.
```

Comment:
if you don't want to implement it now, that's fine, just run multiple expect_success_and_text calls in the test